packages/webapp/pages/_app.tsx

```diff
@@ -131,7 +137,11 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
 
         <link rel="preconnect" href="https://res.cloudinary.com" />
       </Head>
-      <DefaultSeo {...Seo} canonical={canonicalFromRouter(router)} />
+      <DefaultSeo
+        {...Seo}
+        canonical={canonicalFromRouter(router)}
+        titleTemplate={unreadCount === 0 ? undefined : `(${unreadText}) %s`}
```

Comment:
Fix for the issue mentioned on Slack about reseting the title

```suggestion
        titleTemplate={unreadCount ? `(${unreadText}) %s` : '%s' }
```